docs/pages/concepts/logical/relationshipsInfo.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: enhanced
+title: Relationships Information
+permalink: concepts/logical/relationshipsInfo
```

Comment:
From the description of this page, it seems that this page is most appropriately located under Contributing.

There can be two versions of this page that refer to the same relationship data.

@iArchitSharma @vishalvivekm

Reply:
@RipulHandoo, please relocate and rename the page, too, using hyphenation where necessary.